src/components/pages/Home.tsx

```diff
@@ -1,16 +1,57 @@
 import * as React from "react";
+import { Map } from "immutable";
+import { connect, MapStateToProps } from "react-redux";
 
 import { OperationType } from "@raha/api-shared/models/Operation";
-
+import { MemberId } from "@raha/api-shared/models/identifiers";
```

Comment:
I made a breaking change that I'll fix for you when the new `@raha/api-shared` version gets merged, but yeah this will just need to be `@raha/api-shared/dist/...` once #164 is in.